# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1219807618 14400
#      Tue Aug 26 23:26:58 2008 -0400
# Node ID 366821c0c01cb4301b29449981c468cd7b8b266b
# Parent  6333da0dfdfde7945fd8bfc13a36789358042027
Adjust tolerance to single precision test.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2008-08-26 Ben Abbott <bpabott@mac.com>
+
+	* DLD-FUNCTIONS/chol.cc (test): Double tolerance.
+
 2008-08-26  Jaroslav Hajek <highegg@gmail.com>
 
 	* parse.y (Fassignin): Use default  scope when calling varref.
 
 2008-08-26  Michael Goffioul  <michael.goffioul@gmail.com>
 
 	* genprops.awk (emit_declarations): Call updaters/listeners only when
 	'set' method returned true.
diff --git a/src/DLD-FUNCTIONS/chol.cc b/src/DLD-FUNCTIONS/chol.cc
--- a/src/DLD-FUNCTIONS/chol.cc
+++ b/src/DLD-FUNCTIONS/chol.cc
@@ -773,30 +773,30 @@ If @var{info} is not present, an error m
 %! R1 = cholupdate(R,single(u));
 %! 
 %! assert(norm(triu(R1)-R1,Inf) == 0)
 %! assert(norm(R1'*R1 - R'*R - single(u*u'),Inf) < 1e1*eps('single'))
 %! 
 %! R1 = cholupdate(R1,single(u),"-");
 %! 
 %! assert(norm(triu(R1)-R1,Inf) == 0)
-%! assert(norm(R1 - R,Inf) < 1e1*eps('single'))
+%! assert(norm(R1 - R,Inf) < 2e1*eps('single'))
 %! 
 %!test
 %! R = chol(single(Ac));
 %! 
 %! R1 = cholupdate(R,single(uc));
 %! 
 %! assert(norm(triu(R1)-R1,Inf) == 0)
 %! assert(norm(R1'*R1 - R'*R - single(uc*uc'),Inf) < 1e1*eps('single'))
 %! 
 %! R1 = cholupdate(R1,single(uc),"-");
 %! 
 %! assert(norm(triu(R1)-R1,Inf) == 0)
-%! assert(norm(R1 - R,Inf) < 1e1*eps('single'))
+%! assert(norm(R1 - R,Inf) < 2e1*eps('single'))
 */
 
 DEFUN_DLD (cholinsert, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable Function} {[@var{R1}, @var{info}] =} cholinsert (@var{R}, @var{j}, @var{u})\n\
 Given a Cholesky@tie{}factorization of a real symmetric or complex hermitian\n\
 positive definite matrix @w{@var{A} = @var{R}'*@var{R}}, @var{R}@tie{}upper triangular,\n\
 return the QR@tie{}factorization of\n\
